temporalio/ui
# DT-646 - add Reapply Type to Reset

## Description & motivation 💭 
This PR adds a new Select menu in the Workflow Reset form that allows users to specify a Reset Reapply Type. The valid options are "All" (which is the default), "Signal", and "None". This controls which event types are reapplied in the new workflow after the reset point.

see https://docs.temporal.io/tctl-v1/workflow#--reset_reapply_type for more

### Screenshots (if applicable) 📸 
<img width="532" alt="Screen Shot 2023-03-29 at 3 05 43 PM" src="https://user-images.githubusercontent.com/11775628/228677067-feb82642-9051-406b-b088-5d5478e1c015.png">

### Design Considerations 🎨 
N/a - approved by @canvanooo

## Testing 🧪 

### How was this tested 👻 


### Steps for others to test: 🚶🏽‍♂️🚶🏽‍♀️ 
1. On the workflow details page for a running workflow, open the dropdown menu of the workflow actions split button, and click "Reset". 
2. Confirm the "Reapply Type" select menu is visible, and that changing it has an effect on the reapply type that is sent in the request to reset the workflow when the form is submitted.

## Checklists

### Draft Checklist 

### Merge Checklist 

### Issue(s) closed 
closes #1240 
## Docs
Docs Team have been informed of the changes
### Any docs updates needed?
